platform/preferences/preferences-impl/src/main/java/org/codice/ddf/preferences/impl/PreferencesImpl.java

```diff
@@ -221,13 +221,7 @@ public Map<String, Object> get(String userId) throws PreferencesException {
           .filter(Objects::nonNull)
           .forEach(
               attribute -> {
-                String value = attribute.getValue().toString();
```

Comment:
Are we not JSON encoding the attributes being stored in Solr? Above on line 164 we are JSON encoding the value before putting it into Solr.

Reply:
encoding the values during the migration instead